src/mono/wasm/build/WasmApp.Native.targets

```
@@ -160,10 +161,19 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="_SetWasmNativeStripDefault" Condition="'$(WasmNativeStrip)' == ''">
-    <PropertyGroup>
+  <Target Name="_SetDebugSymbolsPropertiesDefaults">
+    <!-- Default with nothing set -->
+    <PropertyGroup Condition="'$(WasmNativeDebugSymbols)' == '' and '$(WasmNativeStrip)' == ''">
       <!-- Build+relink+config=debug -->
-      <WasmNativeStrip Condition="'$(WasmBuildingForNestedPublish)' != 'true' and '$(WasmBuildNative)' == 'true' and '$(Configuration)' == 'Debug'">false</WasmNativeStrip>
+      <WasmNativeDebugSymbols Condition="'$(WasmBuildingForNestedPublish)' != 'true' and '$(WasmBuildNative)' == 'true' and '$(Configuration)' == 'Debug'">true</WasmNativeDebugSymbols>
+
+      <WasmNativeStrip Condition="'$(WasmNativeStrip)' == 'true'">false</WasmNativeStrip>
+      <WasmNativeStrip Condition="'$(WasmNativeStrip)' == 'false'">true</WasmNativeStrip>
```

```suggestion
      <WasmNativeStrip Condition="'$(WasmNativeStrip)' == 'true'">false</WasmNativeStrip>
      <WasmNativeStrip Condition="'$(WasmNativeStrip)' == 'false'">true</WasmNativeStrip>
```
This can only be '' in this PropertyGroup?